ethereum/chain.py

```diff
@@ -263,21 +264,21 @@ def add_block(self, block):
                         break
                     b = self.get_parent(b)
                 replace_from = b.header.number
-                for i in xrange(replace_from, 2**63 - 1):
+                for i in islice(count(), replace_from, 2**63 - 1):
```

Comment:
Yep this bound was included in the original codebase, but I see your point that it won't ever be reached. I updated the code to just use `count()`